vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/bean/boutiquesTools/BoutiquesApplication.java

```diff
@@ -33,6 +33,8 @@ public class BoutiquesApplication implements IsSerializable {
     private Set<BoutiquesOutputFile> outputFiles = new HashSet<>();
     private Map<String, String> tags = new HashMap<>();
     private String jsonFile;
+    private static Set<String> commandLineFlag;
+    private Set<String> customProperties;
```

Comment:
For customProperties, we have the should to store either :
- all the custom properties (but that's anoying because they can have any type)
- store only the vip dot inputs like you do. In this case the field should be named `vipDotInputsIds`

Reply:
renamed the field to vipDotInputIds